### DISCUSSION TITLE
HOOBs Plugin not allowing Link Account

### DISCUSSION THREAD

**Discussion by kelsdunn** [homebridge-simplisafe3/homebridge-simplisafe3#258] [Q&A]
Hi everyone,

I need help with this plugin as there's no way I'm able to enter my credentials for SimpliSafe to establish a connection. After I click the button 'Link Account' on the HOOBS MAC desktop software Plugin configuration section, it continues to say '**Please wait for Bridge to Start**.' After seeing that, I try to start it by going to the respective Bridge tab and hitting 'start' but nothing happens. I'm lost as to how I can move forward...definitely not a pro which is why I am using HOOBS in the first place!

**I looked at the README doc, but I'm still confused!** 

Here is the log report for reference!

```
11/21/2021, 4:44:01 PM Simplisafe3 Bridgehomebridge-simplisafe3.SimpliSafe 3ERRORSimpliSafe login failed with error: Error: No authentication credentials detected.
    at SimpliSafe3AuthenticationManager.refreshCredentials (/var/lib/hoobs/simplisafe3bridge/node_modules/homebridge-simplisafe3/common/authManager.js:191:15)
    at new SS3Platform (/var/lib/hoobs/simplisafe3bridge/node_modules/homebridge-simplisafe3/index.js:84:41)
    at /usr/src/bridge/server.ts:434:46
    at Array.forEach (<anonymous>)
    at Server.loadPlatforms (/usr/src/bridge/server.ts:417:31)
    at Server.start (/usr/src/bridge/server.ts:224:69)
11/21/2021, 4:44:01 PM Simplisafe3 Bridgehomebridge-simplisafe3.SimpliSafe 3ERRORSee the plugin README for more information on authenticating with SimpliSafe.
11/21/2021, 4:44:01 PM Simplisafe3 Bridgehomebridge-simplisafe3.SimpliSafe 3ERRORInitial accessories refresh failed with error: Error: Not authenticated with SimpliSafe.
    at /var/lib/hoobs/simplisafe3bridge/node_modules/homebridge-simplisafe3/index.js:103:56
```

Thanks for the help in advance,
Kelsey

**Comment by shamoon** [comment]
Hi, there have been a few threads for this recently. See #247, #248 and #249, #250, #255 in particular see #249 (comment).

**Comment by wsr554** [comment]
@kelsdunn I have had the same issue and have tried all the options above for weeks and still end up with the same problem. This might be more of a HOOBS issue and I plan to reach out to them as well.  I am new too so it has been frustrating trying to get this to work
<img width="1530" alt="Screen Shot 2021-11-29 at 2 09 10 PM" src="https://user-images.githubusercontent.com/93354391/143953890-26b8af78-3c34-442a-97b7-b6a246cea816.png">
.
